<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" >

<title>GISTIC2.0的安装与使用 | Peer&#39;s blog</title>
<meta name="description" content="书写博客加强学习">

<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">

<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
<link rel="shortcut icon" href="https://peerpoll.github.io//favicon.ico?v=1577152926732">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.10.0/katex.min.css">
<link rel="stylesheet" href="https://peerpoll.github.io//styles/main.css">


  
    <link rel="stylesheet" href="https://unpkg.com/gitalk/dist/gitalk.css" />
  

  


<script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
<script src="https://cdn.bootcss.com/highlight.js/9.12.0/highlight.min.js"></script>

<link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />



  </head>
  <body>
    <div id="app" class="main">

      <div class="sidebar" :class="{ 'full-height': menuVisible }">
  <div class="top-container" data-aos="fade-right">
    <div class="top-header-container">
      <a class="site-title-container" href="https://peerpoll.github.io/">
        <img src="https://peerpoll.github.io//images/avatar.png?v=1577152926732" class="site-logo">
        <h1 class="site-title">Peer&#39;s blog</h1>
      </a>
      <div class="menu-btn" @click="menuVisible = !menuVisible">
        <div class="line"></div>
      </div>
    </div>
    <div>
      
        
          <a href="/" class="site-nav">
            首页
          </a>
        
      
        
          <a href="/archives" class="site-nav">
            归档
          </a>
        
      
        
          <a href="/tags" class="site-nav">
            标签
          </a>
        
      
        
          <a href="/post/about" class="site-nav">
            关于
          </a>
        
      
    </div>
  </div>
  <div class="bottom-container" data-aos="flip-up" data-aos-offset="0">
    <div class="social-container">
      
        
      
        
      
        
      
        
      
        
      
    </div>
    <div class="site-description">
      书写博客加强学习
    </div>
    <div class="site-footer">
      Powered by <a href="https://github.com/getgridea/gridea" target="_blank">Gridea</a> | <a class="rss" href="https://peerpoll.github.io//atom.xml" target="_blank">RSS</a>
    </div>
  </div>
</div>


      <div class="main-container">
        <div class="content-container" data-aos="fade-up">
          <div class="post-detail">
            <h2 class="post-title">GISTIC2.0的安装与使用</h2>
            <div class="post-date">2019-12-24</div>
            
            <div class="post-content">
              <h4 id="因为最近的课题需要将分完的两组数据进行比较其中就有一步是需要比较两组之间的cnv差异需要跑gistic">因为最近的课题需要将分完的两组数据进行比较，其中就有一步是需要比较两组之间的CNV差异，需要跑GISTIC</h4>
<h5 id="gistic的介绍">GISTIC的介绍</h5>
<pre><code>Summary:
The GISTIC module identifies regions of the genome that are significantly amplified or deleted across a set of samples. Each aberration is assigned a G-score that considers the amplitude of the aberration as well as the frequency of its occurrence across samples. False Discovery Rate q-values are then calculated for the aberrant regions, and regions with q-values below a user-defined threshold are considered significant. For each significant region, a “peak region” is identified, which is the part of the aberrant region with greatest amplitude and frequency of alteration. In addition, a “wide peak” is determined using a leave-one-out algorithm to allow for errors in the boundaries in a single sample. The “wide peak” boundaries are more robust for identifying the most likely gene targets in the region. Each significantly aberrant region is also tested to determine whether it results primarily from broad events (longer than half a chromosome arm), focal events, or significant levels of both. The GISTIC module reports the genomic locations and calculated q-values for the aberrant regions. It identifies the samples that exhibit each significant amplification or deletion, and it lists genes found in each “wide peak” region.
References:
Mermel C, Schumacher S, et al. (2011). &quot;GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.&quot; Genome Biology, 12:R41.
Beroukhim R, Mermel C, et al. (2010). &quot;The landscape of somatic copy -number alteration across human cancers.&quot; Nature, 463:899-905.
Beroukhim R, Getz G, et al. (2007). “Assessing the significance of chromosomal abberations in cancer: Methodology and application to glioma.” Proc Natl Acad Sci, 104:20007-20012.
</code></pre>
<h5 id="下载与安装">下载与安装</h5>
<p>下载：</p>
<blockquote>
<p>wget -c ftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz
安装：</p>
</blockquote>
<pre><code>mkdir GISTIC
mv GISTIC_2_0_23.tar.gz GISTIC/
tar zxf GISTIC_2_0_23.tar.gz
</code></pre>
<p>安装Matlab: 因为下载的包中自带安装包所以不需要再次安装</p>
<pre><code>cd MCR_Installer/
unzip MCRInstaller.zip 
./install -mode silent -agreeToLicense yes -destinationFolder 
/root/GISTIC/MATLAB_Compiler_Runtime/
</code></pre>
<p>设置Matlab 变量</p>
<pre><code>echo &quot;export XAPPLRESDIR=/root/GISTIC/MATLAB_Compiler_Runtime/v83/X11/app-defaults:\$XAPPLRESDIR&quot; &gt;&gt; ~/.bashrc
source ~/.bashrc
</code></pre>
<p>运行示例文件，进行测试：</p>
<pre><code>cd /root/GISTIC/
./run_gistic_example
</code></pre>
<h5 id="使用">使用</h5>
<p><a href="https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/CNV_Pipeline/">参考TCGA的参数</a>
所需文件：</p>
<pre><code>TCGA  CNV的segment
#Snp  Snp6.0的芯片注释文件(单独看snp时需要，现在不需要)
hg38  hg38.UCSC.add_miR.160920.refgene.mat
</code></pre>
<p>代码：（注意要切换到GISTIC文件中）</p>
<pre><code>/root/GISTIC/gistic2 -b /root/cluster_1/ -seg /root/clust1_cnv.txt -refgene /root/GISTIC/refgenefiles/hg38.UCSC.add_miR.160920.refgene.mat -ta 0.1 -armpeel 1 -brlen 0.7 -cap 1.5 -conf 0.99 -td 0.1 -genegistic 1 -gcm extreme -js 4 -maxseg 2000 -qvt 0.25 -rx 0 -savegene 1
</code></pre>

            </div>
            
            
              <div class="next-post">
                <div class="next">下一篇</div>
                <a href="https://peerpoll.github.io//post/ji-suan-mian-yi-yi-zhi-xing-de-xiang-guan-fang-fa">
                  <h3 class="post-title">
                    计算肿瘤异质性的相关方法
                  </h3>
                </a>
              </div>
            

            
              
                <div id="gitalk-container" data-aos="fade-in"></div>
              

              
            

          </div>

        </div>
      </div>
    </div>

    <script src="https://unpkg.com/aos@next/dist/aos.js"></script>

<script type="application/javascript">

AOS.init();

hljs.initHighlightingOnLoad()

var app = new Vue({
  el: '#app',
  data: {
    menuVisible: false,
  },
})

</script>



  
    <script src="https://unpkg.com/gitalk/dist/gitalk.min.js"></script>
    <script>

      var gitalk = new Gitalk({
        clientID: 'be71559da69753f62c8a',
        clientSecret: '3cc5d67fce495417f45c14b21452b825f3bdb2e8',
        repo: 'Peerpoll.github.io',
        owner: 'Peerpoll',
        admin: ['Peerpoll'],
        id: (location.pathname).substring(0, 49),      // Ensure uniqueness and length less than 50
        distractionFreeMode: false  // Facebook-like distraction free mode
      })

      gitalk.render('gitalk-container')

    </script>
  

  




  </body>
</html>
